components/List/ListElement.vue

```diff
@@ -47,6 +47,17 @@ export default {
         padding: 8px 12px;
         transition: all 0.3s cubic-bezier(0.25, 0.8, 0.5, 1);
         user-select: none;
+        cursor: move;
+
+        &::before {
```

Comment:
What is that doing ?

Reply:
this is the case with the list and clicking on it, this is the blend that covers the element and click on one element each (problem with the select field list with the checkbox)